### PR TITLE
Feature/add links to footer

### PR DESCRIPTION
**Pull request by positiveimpact**
I added the "impressum" link as well as social media links to the footer.
Additionally I made the footer always stay at the bottom of the page. Before it wasn't of the bottom if the height of the content of the page was smaller than the screen height. 
To accomplish this I had to add an additional div to layout.js and AboutLayout.js. Since every layout needs this div, I decided to create the file LayoutWrapper.js which is used by both layout files. This way we only have to make changes in one place if we want to make any changes to that div in the future.

### CODE REVIEW

File: public/data/projects.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
       "name": "CO2-Labels for University Canteen",
       "image": "/images/canteen.jpg",
       "organisation_name": "sneep Erlangen e.V.",
-      "organisation_image": "/images/sneep.png",
+      "organisation_image": "images/sneep.png",
```
Comment (EvanHahn) [inline public/data/projects.json]: Should this be reverted?

File: public/icons/facebook.svg [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (EvanHahn) [inline public/icons/facebook.svg]: Should we use the [Material UI icon](https://material-ui.com/components/material-icons/) for Facebook instead? The same comment applies for the other icons—Twitter, GitHub, Instagram, etc.

File: src/components/general/Footer.js [hunk 3]
```diff
@@ -1,22 +1,90 @@
 import React from "react";
+import Link from "next/link";
 import { Box } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles(theme => ({
-  root: {
+  footer: {
```
Comment (EvanHahn) [inline src/components/general/Footer.js]: Why did this name change?

File: src/components/layouts/AboutLayout.js [hunk 4]
```diff
@@ -2,32 +2,28 @@ import React from "react";
...
-          className={`${classes.main} ${classes.directlyUnderHeader}`}
-        >
-          {children}
-        </Container>
-        <Footer />
-      </ThemeProvider>
-    </>
+    <LayoutWrapper theme={aboutTheme}>
+      <Header />
+      <Container maxWidth={false} component="main" className={`${classes.main}`}>
```
Comment (EvanHahn) [inline src/components/layouts/AboutLayout.js]: Nitpick: could this just be `className={classes.main}`?

File: src/components/layouts/LayoutWrapper.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Head from "next/head";
+import { makeStyles, ThemeProvider } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  leaveSpaceForFooter: {
+    position: "relative",
+    paddingBottom: 68,
+    minHeight: "100vh"
```
Comment (EvanHahn) [inline src/components/layouts/LayoutWrapper.js]: Nice!

File: src/components/layouts/LayoutWrapper.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Head from "next/head";
+import { makeStyles, ThemeProvider } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  leaveSpaceForFooter: {
+    position: "relative",
+    paddingBottom: 68,
```
Comment (EvanHahn) [inline src/components/layouts/LayoutWrapper.js]: Should this use `theme`?

File: public/data/projects.json (hunk 1 above)
Comment (positiveimpact) [reply public/data/projects.json]: Oh got I keep screwing this up for some reason. Reverted it.

File: public/icons/facebook.svg (hunk 2 above)
Comment (positiveimpact) [reply public/icons/facebook.svg]: Makes sense. Switched to Material UI Icons

File: src/components/general/Footer.js (hunk 3 above)
Comment (positiveimpact) [reply src/components/general/Footer.js]: I built a wrapper around it, removed it again and forgot to change it back. Reverted.

**Comment by positiveimpact** [comment]
Thanks for the review! Implemented all suggested changes:
-added the "/" back in in organisation_image
-switched to material UI icons
-reverted className of root item to "root"
-Fixed the Nitpick
-used theme.spacing for height of footer and padding of body

I also removed some unused CSS that I found in Footer.js

Are there any other remarks? Otherwise I think this is ready to merge.

**Pull request merged**